src/smartMirrorProject/theMirror/views.py

```diff
@@ -276,9 +334,17 @@ def index(request):
             # The codes for the stations can be retrieved with the NS API (https://apiportal.ns.nl/api-details#api=reisinformatie-api&operation=getStations). We've saved
             # an csv file with the station codes in Teams (Algemeen\NS_API)
             "data": fetch_reisplanner(
-                "DID", "AH", 1
+                "AH", "DID", 1
             ),  # vanaf station, naar station, aantal journeys die hij laat zien
```

Comment:
Comments graag in het engels

Reply:
Ik heb hier blijkbaar de verkeerde regel geselecteerd gaat om de regel met de comments eronder

Reply:
Done